app/models/maintenance_tasks/removed_ldap_users.rb

```diff
@@ -32,12 +31,12 @@ def enabled?
   private
 
   def collect_removed_ldap_users
-    User::Human.ldap.collect do |user|
-      user unless ldap_connection.exists?(user.username)
-    end.compact
+    User::Human.ldap.reject do |user|
+      ldap_connection.all_uids.include?([user.username])
```

Comment:
Is this a two dimensional array? Is that why you include on an array? Would it make more sense to flatten it somewhere if there's only the username in the array?

Reply:
Yes you're right I didn't think of that :smile: